Code/Framework/AzToolsFramework/AzToolsFramework/FocusMode/FocusModeSystemComponent.cpp

```diff
@@ -140,7 +163,9 @@ namespace AzToolsFramework
         if (auto iter = AZStd::find(m_focusedEntityIdList.begin(), m_focusedEntityIdList.end(), childId);
             iter != m_focusedEntityIdList.end())
         {
-            m_focusedEntityIdList.erase(iter);
+            // Swap and pop since we don't care about the ordering.
+            *iter = m_focusedEntityIdList.back();
```

Comment:
thought: No need to make this change but just something to mention, using `AZStd::swap` here instead can actually be faster as it's implemented in terms of move, so could be cheaper than actually performing the assignment. In this case as we're just talking about a `u64/EntityId` it won't matter but it can be useful to know for larger types when using the swap/pop technique